server/memphis_handler_schemas.go

```diff
@@ -212,16 +212,18 @@ func (sh SchemasHandler) CreateNewSchema(c *gin.Context) {
 	if updateResults.MatchedCount == 0 {
 		message := "Schema " + schemaName + " has been created"
 		serv.Noticef(message)
+		_, err = schemaVersionCollection.InsertOne(context.TODO(), newSchemaVersion)
```

Comment:
One last thing, the log for creating a new schema should be logged after all steps finished, so move it below the insert version